### DISCUSSION TITLE
Subscription has ended.

### DISCUSSION THREAD

**Discussion by claudiop1975** [community/community#44837] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I was using Copilot in a normal way, and it seemed so good that I decided to do an annual subscription, and apparently, everything was going very well and suddenly it stopped working, the log file gives me this error "[INFO] [auth] [2023 -01-19T22:48:49.282Z] Invalid copilot token: missing token: 403
[ERROR] [default] [2023-01-19T22:48:49.283Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for using GitHub Copilot. Your subscription has ended."" Can someone help me??

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @claudiop1975! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by davecheney** [comment]
@claudiop1975 your CC may have been declined. Please check your billing details and if you believe they are in order, please contact support https://support.github.com/contact
